dotcom-rendering/src/web/components/ArticleContainer.tsx

```diff
@@ -204,16 +161,13 @@ const adVariantCSS = css`
 	}
 `;
 
-export const ArticleContainer = ({ children, format, abTests }: Props) => {
-	const isInline1ContainerSizingVariant =
-		abTests?.inline1ContainerSizingVariant === 'variant';
+export const ArticleContainer = ({ children, format }: Props) => {
```

Comment:
Rather than just omitting the optional property `abTests` could we remove it from `Props` altogether? Or is there a reason to have it hanging around?

Reply:
Thanks good spot!